packages/core/src/shared/date/useCalendarGrid.test.ts

```
@@ -0,0 +1,383 @@
+  })
+})
+
+describe('useCalendarGrid — switching unit', () => {
+  it('rebuilds the grid for the new unit around the same placeholder', async () => {
+    await withScope(async () => {
+      const unit = ref<CalendarUnit>('day')
+      const placeholder = ref(sep5) as any
+      const api = useCalendarGrid({
+        unit,
+        placeholder,
+        setPlaceholder: d => { placeholder.value = d },
```

ESLint

[error] Expected parentheses around arrow function argument having a body with curly braces.

(style/arrow-parens)